mailer.js

```diff
@@ -162,9 +169,11 @@ module.exports = function (log) {
     if (message.redirectTo) { query.redirectTo = message.redirectTo }
     if (message.resume) { query.resume = message.resume }
 
-    var link = this.verificationUrl + '?' + qs.stringify(query)
+    var link = this._generateUTMLink(this.verificationUrl, query, template, 'activate')
+    var alternativeLink = this._generateUTMLink(this.verificationUrl, query, template, 'activate-alternative')
+
     query.one_click = true
-    var oneClickLink = this.verificationUrl + '?' + qs.stringify(query)
+    var oneClickLink = this._generateUTMLink(this.verificationUrl, query, template, 'activate')
```

Comment:
Since this will eventually be a button that is displayed within GMail itself and we'll want to know the click rate of those separately from everything else, should `activate` be `oneclick-activate`?